iv-rules.txt

```diff
@@ -32,6 +32,10 @@ body: //main
 
 # We also fix prev/next links ...
 @detach: //span[has-class("prev")] # same as: @detach: //span[has-class("next")]
+@prepend("Previous page:"): $$
+@prepend("Next page:"): //span[has-class("next")]
```

Comment:
```suggestion
@prepend("←"): $$
@append("→"): //span[has-class("next")]
```

It's better to use something general like arrow marks than hard-coded English words since we have several translations.

Reply:
Speaking of translation, I noticed the IV doesn't work on the localization page. Is that intentional?

Reply:
No. They should all work, and many of them are tested, too. Which page specifically are you talking about?

Reply:
https://grammy.dev/id/guide/introduction.html
https://grammy.dev/es/guide/introduction.html
https://grammy.dev/uk/guide/introduction.html
https://grammy.dev/zh/guide/introduction.html

Reply:
I mean the IV did successfully render the page, but the instant view button in the telegram client does not appear.

Reply:
There is no IV for the direct URLs. You need to use the preview links. The docs bot embeds these links behind invisible characters so it looks like the docs link has an IV page.

Reply:
Oh nvm. Apparently instant view works only when sent by the bot. Would you consider submitting the template to Telegram so it can be rendered globally?

Reply:
We have done that years ago. They have stopped reviewing submissions years before that.